ichnaea/conf.py

```diff
@@ -0,0 +1,117 @@
+"""
+Application settings and configuration derived from the environment
+"""
+from __future__ import absolute_import
+
+import os
+import os.path
+
+from everett.component import ConfigOptions, RequiredConfigMixin
+from everett.manager import ConfigManager, ConfigOSEnv
+
+HERE = os.path.dirname(__file__)
+
+
+class AppConfig(RequiredConfigMixin):
+    required_config = ConfigOptions()
+    required_config.add_option(
+        'testing',
+        default='false',
+        parser=bool,
+        doc='Whether or not we are running tests.'
+    )
+
+    required_config.add_option(
+        'asset_bucket',
+        default='',
+        doc='???'
+    )
+    required_config.add_option(
+        'asset_url',
+        default='',
+        doc='???'
+    )
+
+    # Database related settings
+    required_config.add_option(
+        'db_library',
+        default='pymysql',
+        doc='one of "pymysql" or "mysqlconnector"'
```

Comment:
Ichnaea uses one in the webapp because it's gevent friendly and the other in other cases. I think unless there are compelling reasons to do otherwise, we should just standardize on one and not switch back and forth. That'll simplify a few things.

I have vague memories of this coming up before, but I forget whether we decided anything.

Reply:
My memories are also vague. It may be tied up in using a connection pool, which is good for read-only webapp connections, and a direct connection, which is better for DDL changes. Or, I may be remembering past struggles with schema changes and SQLAlchemy.

There was quite a bit of drama around Python MySQL libraries, including the Python 2 to 3 transition. I think this has mostly shaked out by 2019. I'm inclined to standardize on ``mysqlclient``, the [Django recommendation](https://docs.djangoproject.com/en/2.2/ref/databases/#mysql-db-api-drivers). I'm also inclined to investigate a switch to PostgreSQL. All of this is out of scope for this PR though.

Reply:
I pushed off the "let's switch to postgres" to another day. I like the idea of revisiting the mysql client library, so I wrote up #870 for that.